netlink.go

```diff
@@ -0,0 +1,96 @@
+package cyw43439
+
+import (
+	"net"
+
+	"tinygo.org/x/drivers/netlink"
+)
+
+func (d *Device) GetHardwareAddr() (net.HardwareAddr, error) {
+	return net.HardwareAddr(d.mac[:6]), nil
+}
```

Comment:
I much prefer `MACAs6`. It goes back to the `netip` changes to `net` package's `netdev` interface. This is part of a redesign I've been wanting for the netdev/netlink interface which is pending analysis on my end

Reply:
Do you want:

`    return net.HardwareAddr(d.MACAs6()[:]), nil
`

?